exercises/sub-list/SublistExample.swift

```diff
@@ -0,0 +1,72 @@
+import Foundation
+
+enum SublistKind: String {
```

Comment:
Yes the raw value is unused,but I added it since the expected values for each test case shown in  
https://github.com/exercism/x-common/blob/master/exercises/sublist/canonical-data.json are given as strings so I thought we could extract the raw values if required.Anyways,I've removed the String raw value for SublistKind in my latest commit.